### ISSUE TITLE
Maybe make the repetition a bit more clear

### ISSUE THREAD

**Issue by fflorent** [cozy/cozy-calendar#346]
First of all, I am really glad of the options for the repetition. That's *really* awesome we can setup a repetition "every 2 months on the 4th Friday" (for example). 

I just have a little feedback on the obviousness for configuring that. If you want to configure the repetition, you have these options:

![repetition list](https://cloud.githubusercontent.com/assets/371705/8895865/281255ee-33e9-11e5-9311-82d907215ac3.png)

When I first saw that, I thought we couldn't configure much the repetition except once per week, once per day, etc. I wonder if there is no opportunity to make these items clearer. I was thinking of something like : 
  * "no repeat"
  * "once per X days"
  * "once per X weeks"
  * ...

Though I won't be surprised if you find something smarter :).

Florent

**Comment by jsilvestre** [comment]
Those labels would not be clearer in my humble opinion, because you can do more than that by tweaking various options. I'm going to ask people around if they have good ideas. If you think we can improve it, there is a good chance that we can, but I don't have ideas right now.

**Comment by Thatoo** [comment]
If you want inspiration, I found Thunderbird very clear and extremely complete on this topic.

**Comment by fflorent** [comment]
Yep, Lightning for Thunderbird has something clear (a "custom" item in which you can go deeper in configuration). I can't attach screenshots, weirdly, but I can provide them once I am at home.

Florent

**Comment by f6fvy** [comment]
I guess Florent did a mistake by attaching the reminder dialogs, but the repeat dialogs of Thunderbird / Lightning are IMHO the way to go.
![repeat_1](https://cloud.githubusercontent.com/assets/12431844/8909141/21144cd8-3480-11e5-8240-ff3227192600.png)
![repeat_2](https://cloud.githubusercontent.com/assets/12431844/8909148/25151bbe-3480-11e5-90b8-345ed522fa4e.png)
![repeat_3](https://cloud.githubusercontent.com/assets/12431844/8909151/277e6928-3480-11e5-96d5-a6f8149fd52e.png)

**Comment by fflorent** [comment]
Hmm, yeah... Though the idea is the same :)
